tests/Microsoft.Identity.Test.Unit/TelemetryTests/RegionalTelemetryTests.cs

```diff
@@ -75,7 +75,7 @@ public async Task TelemetryAcceptanceTestAsync()
 
             // we can assert telemetry here, as it will be sent to AAD. However, AAD is down, so it will not record it.
             AssertCurrentTelemetry(result.HttpRequest, ApiIds.AcquireTokenForClient,
-                RegionAutodetectionSource.Cache.ToString("D"),
+                RegionAutodetectionSource.EnvVariable.ToString("D"),
```

Comment:
We should not change existing behavior. If region auto-detection completed, we should reuse that result.

Reply:
Let us discuss this tomorrow.